terraform/monitoring/panels/app/publishing_workers_errors.libsonnet

```diff
@@ -11,9 +11,30 @@ local targets   = grafana.targets;
       datasource  = ds.prometheus,
     )
     .configure(defaults.configuration.timeseries)
+
+    .setAlert(vars.environment, grafana.alert.new(
+      namespace     = vars.namespace,
+      name          = '%(env)s - Notify publisher worker task error' % { env: vars.environment },
+      message       = '%(env)s - Notify publisher worker task error' % { env: vars.environment },
+      notifications = vars.notifications,
+      noDataState   = 'no_data',
+      period        = '0m',
+      conditions    = [
+        grafana.alertCondition.new(
+          evaluatorParams = [ 0 ],
+          evaluatorType   = 'gt',
+          operatorType    = 'or',
+          queryRefId      = 'PublishingWorkersErrors',
+          queryTimeStart  = '1m',
+          queryTimeEnd    = 'now',
+          reducerType     = grafana.alert_reducers.Avg
+        ),
+      ],
+    ))
+
     .addTarget(targets.prometheus(
       datasource  = ds.prometheus,
-      expr        = 'sum(rate(publishing_workers_errors_total{}[$__rate_interval]))',
-      refId       = "availability",
+      expr        = 'sum(rate(publishing_workers_errors_total{}[$__rate_interval])) or vector(0)',
```

Comment:
My only concern here is this doesn't allow us to catch actual no data scenarios.

Interesting, I don't need the `or vector(0)` for the `relay_outgoing_message_failures` or `relay_incoming_messages_total` metrics, even though they are both counters and there are periods of time where no failures happen, but it still reads 0. What's the difference here, and can we copy this instead?